### PR TITLE
Feature/display svg

### PR DESCRIPTION
**Pull request by sjumba**
I have created a component that displays svg images through dataurl.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
+import React, { Component } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import chooseFolder from "../src/components/chooseFolder/chooseFolder";
+import displaySvg from "../src/components/displaySvg/displaySvg";
+class App extends Component {
+  render() {
+    return (
+      <div className="app">
+        <BrowserRouter>
+          <div className="h-100">
```
[the remark this answers is not in the document]
Comment (sjumba) [reply src/App.js]: It was just there incase I ma want to use it, but its deleted now

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.App {
+  text-align: center;
+}
```
[the remark this answers is not in the document]
Comment (sjumba) [reply src/App.css]: done

File: src/components/displaySvg/displaySvg.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+class DisplaySvg extends Component {
+  render() {
+    return (
+      <div className="App">
+        <div className="svg-item">
```
[the remark this answers is not in the document]
Comment (sjumba) [reply src/components/displaySvg/displaySvg.js]: It was just there incase I ma want to use it, but its deleted now

File: src/components/displaySvg/displaySvg.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+class DisplaySvg extends Component {
+  render() {
+    return (
+      <div className="App">
```
[the remark this answers is not in the document]
Comment (sjumba) [reply src/components/displaySvg/displaySvg.js]: Removed

File: src/index.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.scss';
+import './index.css';
```
Comment (CastelloDev) [inline src/index.js]: change this back scss please

File: src/App.js [hunk 6]
```diff
@@ -1,26 +1,22 @@
...
-    </div>
-  );
+import React, { Component } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import chooseFolder from "../src/components/chooseFolder/chooseFolder";
+import displaySvg from "../src/components/displaySvg/displaySvg";
+class App extends Component {
+  render() {
+    return (
+      <div className="app">
```
Comment (CastelloDev) [inline src/App.js]: where is this classname defined

File: src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (CastelloDev) [reply src/App.js]: looks like its still there to me

File: src/App.js (hunk 6 above)
Comment (sjumba) [reply src/App.js]: removed

File: src/components/displaySvg/displaySvg.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+class DisplaySvg extends Component {
+  render() {
+    return (
+      <div>
+        <div>
```
Comment (CastelloDev) [inline src/components/displaySvg/displaySvg.js]: whats the purpose of the second div

**Pull request merged**